diff --git a/media/ozone/media_ozone_platform.cc b/media/ozone/media_ozone_platform.cc
index dcb1078..444192a 100644
--- a/media/ozone/media_ozone_platform.cc
+++ b/media/ozone/media_ozone_platform.cc
@@ -48,6 +48,10 @@ MediaOzonePlatform* CreateMediaOzonePlatformTest() {
   return new MediaOzonePlatformStub;
 }
 
+MediaOzonePlatform* CreateMediaOzonePlatformFb() {
+  return new MediaOzonePlatformStub;
+}
+
 MediaOzonePlatform::MediaOzonePlatform() {
   CHECK(!instance_) << "There should only be a single MediaOzonePlatform.";
   instance_ = this;
